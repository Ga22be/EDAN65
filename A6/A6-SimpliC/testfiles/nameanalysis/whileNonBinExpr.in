int a() {
    while (1 == 1)  {
        return 1;
    }

    while (1 + 1) {
        return 2;
    }
}