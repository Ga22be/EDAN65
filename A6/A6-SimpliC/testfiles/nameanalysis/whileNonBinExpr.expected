Error at line 6: Expected boolean, got: IntType
